apps/rush/src/utilities/Telemetry.ts

```diff
@@ -3,34 +3,43 @@
 
 import * as path from 'path';
 import * as fsx from 'fs-extra';
+import { cloneDeep } from 'lodash';
 
 import {
-  RushConfiguration
+  RushConfiguration,
+  rushVersion
 } from '@microsoft/rush-lib';
 
 export interface ITelemetryData {
   name: string;
   duration: number;
   result: string;
-  timestamp: number;
+  timestamp?: number;
+  platform?: string;
+  rushVersion?: string;
   extraData?: { [key: string]: string };
 }
 
 export default class Telemetry {
+  private readonly folderName: string = 'telemetry';
```

Comment:
I am moving it back to a local const. It will be promoted to rush-lib when it gets out of alpha mode. 

---
In reply to: [118327537](https://github.com/Microsoft/web-build-tools/pull/214#discussion_r118327537) [](ancestors = 118327537)